var/spack/repos/builtin/packages/py-datalad/package.py

```diff
@@ -97,24 +106,31 @@ class PyDatalad(PythonPackage):
         depends_on("py-vcrpy", type=("build", "run"))
```

Comment:
I don't see a version requirement on many of these test requirements in the setup.py, did that change?

Reply:
You are right. This is fixed now.